### PR TITLE
Improve direct assessment result workflow

### PR DESCRIPTION
**Pull request by derekprior**
* Add tabs to all pages
* Internationalize text
* Hide actions from users not authorized for them
* Improve styling

### CODE REVIEW

File: app/views/subjects/show.html.erb [hunk 1]
```diff
@@ -1,23 +1,28 @@
-<h1>Direct Assessments for <%= @subject %></h1>
-<% if @subject.direct_assessments.any? %>
+<% tab(TabHelper::DATA_ENTRY) %>
+
+<h1><%= @subject %></h1>
+
+<% if @subject.direct_assessments.present? %>
```
Comment (joshuaclayton) [inline app/views/subjects/show.html.erb]: `#any?` instead of `#present?`?

File: spec/features/user_creates_a_result_spec.rb [hunk 2]
```diff
@@ -77,20 +60,4 @@
     expect(page).to have_content "Annual survey"
     expect(page).to have_content "92"
   end
-
-  scenario "and it defaults to the same name, description as the assessment" do
```
Comment (joshuaclayton) [inline spec/features/user_creates_a_result_spec.rb]: are these tests duplicated somewhere else?

File: app/views/subjects/show.html.erb (hunk 1 above)
Comment (derekprior) [inline app/views/subjects/show.html.erb]: That will trigger an exists query, which if there are any, will end up doing another round trip when I iterate the collection below. `present?` will trigger one query with either all of my data or an empty list.

File: spec/features/user_creates_a_result_spec.rb (hunk 2 above)
Comment (derekprior) [inline spec/features/user_creates_a_result_spec.rb]: I thought they were, but looking at it, they are not. I can do it more directly with a controller spec, though.

Comment (mrgilman) [inline spec/features/user_creates_a_result_spec.rb]: I think we need these tests to ensure that the results form is pre-filled with the default attributes from the assessment.

Comment (derekprior) [inline spec/features/user_creates_a_result_spec.rb]: I added these expectations back to the existing tests.

**Pull request closed without merging**